frame/elections-phragmen/src/mock.rs

```diff
@@ -0,0 +1,49 @@
+use frame_support::{dispatch::Weight, weights::constants::RocksDbWeight};
```

Comment:
ahh sorry I thought this pallet had a mock file 🤦, I meant inside the rest of the mock stuff (inside `#[cfg(test)] mod test {...}`) but it is okay.